src/app/page.tsx

```
@@ -1,103 +1,93 @@
-import Image from "next/image";
+import MaxWidthWrapper from "@/components/MaxWidthWrapper";
+import { Button, buttonVariants } from "@/components/ui/button";
+import Link from "next/link";
+
+import {
+  Headphones,
+  Infinity,
```

Biome (2.1.2)

[error] Do not shadow the global "Infinity" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)